### ISSUE TITLE
Component text name search doesn't find children of KeepAlive

### ISSUE THREAD

**Issue by jacobg** [vuejs/devtools#601]
But if selecting component via dom, it does pull it up.

**Comment by alexzhang1030** [comment]
Hi, I'm not reproduce it, can you please provide a minimal reproduction? Thanks

**Comment by alexzhang1030** [comment]
Are you using functional component?

**Comment by jacobg** [comment]
No, I don't think so. We do use RouterView, KeepAlive, and <component :is=""">. I'll have to work on an MRE.
